packages/plugins/plugin-mixpanel/src/methods/__mocks__/mockIntegrationSettings.ts

```diff
@@ -0,0 +1,18 @@
+export const mockIntegrationSettings = {
+  integrations: {
+    Mixpanel: {
+      token: '1234',
+      enableEuropeanEndpoint: false,
+      eventIncrements: [],
+      groupIdentiferTraits: [],
+      people: true,
+      peopleProperties: [],
+      propIncrements: ['prop1', 'prop2', 'prop3'],
+      setAllTraitsByDefault: false,
+      superProperties: [],
+      trackAllPages: false,
+      trackCategorizedPages: false,
+      trackNamedPages: false,
+    },
+  },
+};
```

Comment:
Also rename to `sampleIntegrationSettings` or something like that, since it is not a mock per se

Reply:
changed the name and moved it into a different folder under `__mocks__` because I can't figure out how to get jest to ignore it in the `__tests__` folder.

Reply:
Ah, put it in `_tests__/__helpers__`

Reply:
yeah i did that and it still failed😅